docs/features/features-walk-through/notifications.md

```diff
@@ -0,0 +1,14 @@
+# Notifications
+
+Notifications are sent at specific events (e.g. community invitation, community recod submission), to inform users of relevant actions.
+For instance customization possibilities, see [customize notifications](../../customize/notifications.md)
+
+## Notification Settings
+
+Available notification settings can be accessed via the account settings page (`<domain>/account/settings/notifications/`). Additionally, the main menu has an entry for easy access:
+
+![Notifications Setting Menu Entry](./img/notifications/menu.png "Notification Settings Menu Entry")
+
+For personal preferences, it is possible to completely enable or disable receiving notifications. The info on the right hand side will show the channel on which notifications will be received.
```

Comment:
// Suggestion: Include a config variable for users to set notifications on by default, overriding the current 'off' default setting.